javascript-bank/bank.js

```diff
@@ -0,0 +1,41 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+
+  if (balance > 0 && balance % 1 === 0) {
+    var newAccount = new Account(this.nextAccountNumber, holder);
+    newAccount.deposit(balance);
+    this.accounts.push(newAccount);
+    this.nextAccountNumber++;
+    return this.nextAccountNumber - 1;
```

Comment:
Here it would be better to get the account number directly from the `newAccount`.

```JAVASCRIPT
return newAccount.number;
```